build.sbt

```diff
@@ -406,7 +406,6 @@ lazy val compiler = configureAsSubproject(project)
         (products in Compile in packageBin in LocalProject("interactive")).value ++
         (products in Compile in packageBin in LocalProject("scaladoc")).value ++
         (products in Compile in packageBin in LocalProject("repl")).value ++
-        (products in Compile in packageBin in LocalProject("repl-jline")).value ++
```

Comment:
Why remove `repl-jline`? We want to be able use both, shadowed and non-shadowed JLine versions, so we need both projects.